ebike_city_tools/optimize/round_optimized_sort_selection.py

```diff
@@ -0,0 +1,224 @@
+import math
+import pandas as pd
+import networkx as nx
+import numpy as np
+
+from ebike_city_tools.optimize.linear_program import define_IP
+from ebike_city_tools.optimize.round_simple import edge_to_source_target
+from ebike_city_tools.utils import (
+    lane_to_street_graph,
+    extend_od_circular,
+    compute_car_time,
+    compute_edgedependent_bike_time,
+    output_to_dataframe,
+)
+from ebike_city_tools.iterative_algorithms import transform_car_to_bike_edge
+from ebike_city_tools.metrics import compute_travel_times_in_graph
+
+FLOW_CONSTANT = 1
+
+def sort_by_bikevalue(capacities) :
+    """Sorts the capacities by planned bike capacity, with ties broken by car capacity."""
+    return capacities.sort_values(["u_b(e)", "u_c(e)"], ascending=[False, True])
+
+def rounding_error(fractional_value):
+    """Given a fractional value, returns the distance to the closest integer."""
+    next_int = round(fractional_value)
+    if fractional_value >= 0:
```

Comment:
I don't understand this line - the fractional_value should always be greater 0, shouldn't it? So this condition is usually true, so the `rounding_error` method usually yields 0?